blog/_posts/2017-09-23-RGSoC- the grand finale.md

```diff
@@ -0,0 +1,99 @@
+
+The whole journey was super awesome. Starting from day 1 of the event kickoff to this moment, we have learned a lot. Working with our amazing panel of coaches, mentor and supervisor and interacting with the RGSoC community has been a thoroughly enriching experience.
+
+We have been working on  [AFDC](https://leagues.afdc.com/) ( Atlanta Flying Disc Club League Management System), which is a Ruby on Rails application used by the Atlanta Flying Disc Club to manage league registrations, rosters, schedules, and result tracking. The application runs on docker platform.
+
+## Workspace:
+ We had a super cool workplace, the [FOSS](http://foss.amrita.ac.in/) lab in our college where we settle down in the evening (soon after our classes) and continue working till late night. And of course, the coding continues until midnight. We are really grateful to our whole FOSS team and to our friends, who supported us all these days.
+
+<img src="/img/blog/2017/workspace.jpg" alt="Our workspace"/>
+<div align="center" class="image-credits">Our Workspace- the FOSS lab(img credits: Anagha's phone)</div>
+
+
+
+## Tools:
+ The collaborative platform 'Git', the task manager ' Trello ' and our regular team meetings on Hangouts and the most important one, our day-to-day discussion space, Slack. These kept us well connected throughout the journey.
+
+## Things learnt:
+
+* Learnt to work with docker.
+
+* Learnt Git basics and a lot more.
+
+* Worked on more features in Ruby on Rails.
+
+* Got a good grip in Ruby.
+
+* Learnt a lot of new terms and concepts.
+
+* Time management.
+
+* Patience.
+
+* Being regular with updates.
+
+## Issues worked on:
+
+This summer taught us a lot about the workflow of a project and how priorities change. We started with three issues in the beginning of the summer and now, towards the end, our plan has more or less changed. We integrated the site with it's MailChimp account. This feature interested us a lot because we experimented around with MailChimp and actually sent out a few promotional e-mails(from a dummy account). Now we know how all the popular websites send out all those emails :) 
+Implementing it took a lot of time because with every single line of code we wrote we learnt so much. 
+
+We also took up the job of porting the project to Bootstrap 3 and making the pages mobile-responsive. That helped us go through the whole project page-by-page giving us a lot of insight into how a Rails application works. 
+
+In the midst of this, we solved many minor issues in the repo. 
+We lodged a few issues, cleaned up the documentation by relating to the minor difficulties we had in setting up the project on our systems.  
+
+Our next major feature is to implement an Attendance system. Still in the process and loving every moment of it. 
+
+## Problems faced:
+
+<img src="/img/blog/2017/confused.gif" alt="Thankyou"/>
+<div align="center" class="image-credits">credits:giphy.com</div>
```

Comment:
Add a space between 'credits' and 'giphy.com'. Follow the same style for next 2 images/giphy for the consistency purpose in a single blog post. :)